spec/presenters/school_vacancies_presenter_spec.rb

```diff
@@ -18,78 +18,86 @@
   end
   let!(:published_vacancies) { create_list(:vacancy, 7, :published, school: school) }
 
+  let(:vacancy_ids) do
+    vacancies.sort_by { |vacancy| vacancy['job_title'].delete(' ') }.reverse.pluck(:id)
+  end
+
```

Comment:
This is specifically for when it's sorted by `column: 'job_title', order: 'desc'`, and isn't general. I'm not sure it makes sense to pull out to here.

Reply:
Ah, well spotted. I just got refactor-happy there :D

Reply:
I've wrapped the sorting stuff in a `describe` block to be clearer about what we're doing here